fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/LuceneIndexTest.java

```diff
@@ -4737,6 +4738,132 @@ void testAutoCompleteSearchForPhraseWithoutFreqsAndPositions(IndexedType indexed
         }
     }
 
+    protected static final List<String> autoCompleteCJKPhrases = List.of(
```

Comment:
It seems like we could use a fair amount more testing around these analyzers, particularly around the indexing half, but this seems like an acceptable start.